src/App.js

```diff
@@ -79,20 +107,64 @@ class App extends React.Component {
   renderBody() {
     if (this.state.authInitializing) {
       return null;
+    } else if (this.state.newuser===true){
+      let contactmessage = "";
+      if (this.state.organization_url!=="" && this.state.organization_name!==""){
+        contactmessage = "Contact <a href="+this.state.organization_url+">"+this.state.organization_name+"</a> to get access";
+      }else if (this.state.organization_name!==""){
+        contactmessage = "Contact "+this.state.organization_name+" to get access";
+      }
+      return (
+        <div className = "body-container">
+          <div className = "page-container centralize logged-out">
+            <br />
+            <h2>Interested to join us?</h2>
+            <br /><br />
+            {this.state.newuseremail} is currently not associated with an account.
+            <br />{contactmessage}
+            <br />
+            <br />
+            <span style={{margin:8}}>
+              <Button href={WOOLAROO_URL} variant="outlined" color="secondary">
+               Back to Woolaroo
+              </Button>
+            </span>
+            
+            <span style={{margin:8}}>
+              <Button variant="contained" 
+                color="secondary"
+                onClick={() => this.authAction_()}
+              >
+               Try again
+              </Button>
+            </span>
+          </div>
+        </div>
+      );
     } else if (!this.state.email) {
       return (
         <div className = "body-container">
-          <div className = "page-container logged-out">
+          <div className = "page-container centralize logged-out">
+        
             <img
               src={LANDING_IMG}
               alt=""
               className="landing-page-img"
             />
+            <br /><br />
+            <div style={{display:'inline-block',maxWidth:500}}>Woolaroo is an open-source Google platform that helps indigenous language organizations build their own photo-translation web-apps
```

Comment:
nit: instead of using inline style, add a class and move this styling to the alongside css file. I see this in other parts of the PR as well, would be nice to fix them all before merging.